### PR TITLE
Cloudera package removal

### PR DESCRIPTION
**Pull request by szvasas**

### CODE REVIEW

File: src/test/com/cloudera/sqoop/lib/TestFieldFormatter.java [hunk 1]
```diff
@@ -19,6 +19,8 @@
 package com.cloudera.sqoop.lib;
 
 import org.junit.Test;
+import org.apache.sqoop.lib.FieldFormatter;
+
```
Comment (ebogi) [inline src/test/com/cloudera/sqoop/lib/TestFieldFormatter.java]: Could you please remove this unnecessary line?

File: src/java/org/apache/sqoop/mapreduce/db/netezza/NetezzaExternalTableImportMapper.java [hunk 2]
```diff
@@ -40,18 +40,17 @@
...
 import org.apache.sqoop.util.PerfCounters;
 import org.apache.sqoop.util.TaskId;
 
-import com.cloudera.sqoop.mapreduce.AutoProgressMapper;
-
 /**
  * Netezza import mapper using external tables.
  */
 public abstract class NetezzaExternalTableImportMapper<K, V> extends
-  AutoProgressMapper<Integer, NullWritable, K, V> {
```
Comment (ebogi) [inline src/java/org/apache/sqoop/mapreduce/db/netezza/NetezzaExternalTableImportMapper.java]: This seems to be only a reformatting change, could you please revert it?

File: src/java/com/cloudera/sqoop/tool/ToolDesc.java [hunk 3]
```diff
@@ -17,6 +17,9 @@
  */
 package com.cloudera.sqoop.tool;
 
+
```
Comment (ebogi) [inline src/java/com/cloudera/sqoop/tool/ToolDesc.java]: Could you please remove this unnecessary new line?

File: src/test/org/apache/sqoop/orm/TestParseMethods.java [hunk 4]
```diff
@@ -30,16 +30,13 @@
 import org.apache.hadoop.mapred.FileOutputFormat;
 import org.apache.hadoop.mapred.JobClient;
 import org.apache.hadoop.mapred.JobConf;
-import org.apache.sqoop.testutil.BaseSqoopTestCase;
-import org.apache.sqoop.testutil.CommonArgs;
-import org.apache.sqoop.testutil.HsqldbTestServer;
+import org.apache.sqoop.testutil.*;
```
Comment (ebogi) [inline src/test/org/apache/sqoop/orm/TestParseMethods.java]: Please avoid wild card import usage.

File: src/test/com/cloudera/sqoop/TestMultiMaps.java [hunk 5]
```diff
@@ -36,10 +35,7 @@
 import org.apache.sqoop.SqoopOptions;
 import org.apache.sqoop.SqoopOptions.InvalidOptionsException;
 import org.apache.sqoop.orm.CompilationManager;
-import org.apache.sqoop.testutil.BaseSqoopTestCase;
-import org.apache.sqoop.testutil.CommonArgs;
-import org.apache.sqoop.testutil.HsqldbTestServer;
-import org.apache.sqoop.testutil.ImportJobTestCase;
+import org.apache.sqoop.testutil.*;
```
Comment (ebogi) [inline src/test/com/cloudera/sqoop/TestMultiMaps.java:38]: Please avoid wild card import usage.

File: src/test/org/apache/sqoop/manager/TestSqlManager.java [hunk 6]
```diff
@@ -18,19 +18,237 @@
...
+import org.apache.commons.logging.LogFactory;
+import org.apache.sqoop.SqoopOptions;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
-import org.apache.sqoop.SqoopOptions;
+import org.apache.sqoop.testutil.HsqldbTestServer;
+
+import static org.junit.Assert.*;
```
Comment (ebogi) [inline src/test/org/apache/sqoop/manager/TestSqlManager.java]: Please avoid wild card import usage.

File: src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.sqoop.hive;
+
+import static org.mockito.Mockito.*;
```
Comment (ebogi) [inline src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java:20]: Please avoid wild card import usage.

File: src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+import org.apache.sqoop.SqoopOptions;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.sql.*;
```
Comment (ebogi) [inline src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java:32]: Please avoid wild card import usage.

**Review by ebogi — COMMENTED** [review]
Hi Szabi, I have added some comments to your change otherwise it looks good so far.

File: src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java (hunk 7 above)
Comment (szvasas) [reply src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java:20]: TestTableDefWriterForExternalTable is just a renamed version of org.apache.sqoop.hive.TestTableDefWriter, the wild card import was in the original version too so I would leave it like this for now.

File: src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java (hunk 8 above)
Comment (szvasas) [reply src/test/org/apache/sqoop/hive/TestTableDefWriterForExternalTable.java:32]: TestTableDefWriterForExternalTable is just a renamed version of org.apache.sqoop.hive.TestTableDefWriter, the wild card import was in the original version too so I would leave it like this for now.

**Pull request closed without merging**